src/ert_storage/database_schema/metadatafield.py

```diff
@@ -3,10 +3,4 @@
 
 
 class MetadataField:
-    _metadata = sa.Column("metadata", sa.JSON, nullable=True)
-
-    @property
-    def metadata_dict(self) -> Mapping[str, Any]:
-        if self._metadata is None:
-            return dict()
-        return self._metadata
+    metadata_ = sa.Column("metadata", sa.JSON, nullable=False, default=dict)
```

Comment:
I sort still prefer to have `_` in front, or then change name completely like `datainfo`  :)